giraffe/src/components/SimpleTable/PagedTable.tsx

```diff
@@ -54,31 +56,28 @@ const getNumberOfRowsOnCurrentPage = (
   let lastSignature
   let signature
 
-  // rowHeight is now guaranteed to be greater than zero
-  const lastVisibleRowMinimumHeight = 0.2 * rowHeight
-
```

Comment:
`lastVisibleRowMinimumHeight` was not accurately descriptively of what this value was actually doing. This value was an _additional_ margin of space on top of the space already allocated to the next row. We don't need this.

Previously, we put in this value because we didn't realize the `signature` (see below) was not accurately detecting new table headers, which are taller than a row.